setup.py

```diff
@@ -46,7 +46,9 @@ def find_version(*file_paths):
     'cylc-flow>=8.0b0',
     'jupyterhub==1.3.*',
     'tornado==6.1.*',
+    'traitlets>=5.0.5',
     'graphene-tornado==2.6.*',
+    'graphene>=2.1.8',
```

Comment:
I think we can both move the order, and leave without any version range. Just `traitlets` and `graphene`? That way the version installed is always the version specified in the upstream dependency.

Reply:
OK I finally understand.  I thought @oliver-sanders comment depended on these being transitive dependencies, which they aren't, but I guess our own light use of them is unlikely to be version-dependent.

Reply:
Done.